src/Sulu/Bundle/PageBundle/Resources/js/containers/Form/fields/PageSettingsNavigationSelect.js

```diff
@@ -41,7 +26,7 @@ class PageSettingsNavigationSelect extends React.Component<FieldTypeProps<Array<
                 onChange={this.handleChange}
                 values={value || []}
             >
-                {this.webspace.navigations.map(({key, title}) => (
+                {webspaceStore.getWebspace(formInspector.options.webspace).navigations.map(({key, title}) => (
```

Comment:
same as above `{webspaceStore.getWebspace(formInspector.options.webspace).navigations` could be a `@computed` value

Reply:
same as above would move to `@computed` if render to avoid this on every render call